suggestions/rss_readers.json

```diff
@@ -4,5 +4,19 @@
         "label": "Feeder",
         "source": "fg",
         "repo": "https://gitlab.com/spacecowboy/Feeder"
+    },
+    {
+        "id": "me.ash.reader",
+        "label": "Read You",
+        "source": "f",
+        "reason": "A modern and elegant RSS reader with Material You design",
+        "repo": "https://github.com/Ashinch/ReadYou"
+    },
+    {
+        "id": "com.nunti",
+        "label": "Nunti",
+        "source": "f",
+        "reason": "A smart RSS reader that learns from your preferences to offer you better article suggestions",
+        "repo": "https://gitlab.com/ondrejfoltyn/nunti"
```

Comment:
Requires a review. @shuvashish76: Any opinions?

Reply:
- React-native app, heavy size
- Nunti uses system webview vs Feeder phrases web texts/images into the app.
- The welcome wizard suggests _Topics_ & _News in other languages_. @DodoLeDev What's the source of their suggestions?

Reply:
Nunti has the unique feature of trying to suggest what you should be most interested in, among all the articles provided by your RSS feeds.
The source of the suggestions comes from [the community and the developer](https://gitlab.com/ondrejfoltyn/nunti/-/blob/master/Nunti/src/DefaultTopics.ts).

Reply:
[Adaptive learning](https://gitlab.com/ondrejfoltyn/nunti/-/issues/28#note_874896901), yes.

Reply:
> Nunti has the unique feature of trying to suggest what you should be most interested in, among all the articles provided by your RSS feeds.

> [Adaptive learning](https://gitlab.com/ondrejfoltyn/nunti/-/issues/28#note_874896901), yes.

It says how it works, it does not say whether it actually works.